[EDU-2735] refactor: Update Next.js Static Boilerplate guide (#450)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/templates/nextjs-static-boilerplate/nextjs-static-boilerplate.mdx

---

## About static exports

> The core of Next.js supports static exports. This allows you to initiate your project either as a static site or a SPA and then add features that require a server. By using these more advanced features, Next.js generates an HTML file per route, breaking a strict SPA into individual HTML files, and your project can be deployed and hosted on any web server that can serve HTML/CSS/JS static assets.

Reviewer: ```suggestion
The core of Next.js supports static exports. This allows you to initiate your project either as a static site or a SPA and then add features that require a server. By using these advanced features, Next.js generates one HTML file per route, breaking a strict SPA into individual HTML files, and your project can be deployed and hosted on any web server that can serve HTML/CSS/JS static assets.
```
